.upstream-tests/test/std/utilities/utilities.general/broken_constant_eval.pass.cpp

```diff
@@ -11,7 +11,7 @@
 // UNSUPPORTED: windows, icc, pgi
 // XFAIL: clang-9 && c++11
 // XFAIL: clang-10 && c++11
-// XFAIL: gcc-9 && c++11
+// gcc 10.0 is expected to pass, but later versions do not.
```

Comment:
Is this meant to say GCC 9?

Reply:
See the list in the issue. I might have an inverted sense of what counts as passing because it's an expected failure.

Regardless, 10.0 is not susceptible while 10.1 and 10.2 are.

Reply:
I see. 10.0 is not a released version, so to avoid confusion we should probably not refer to it.